src/components/InputBox/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { SetStateAction, useState } from 'react';
+import EyeIcon from '../../../public/assets/icons/eye';
+import EyeCloseIcon from '../../../public/assets/icons/eyeClose';
+
+type Props = {
+    type: string;
+    name: string;
+    onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    placeholder?: string;
+    required: boolean;
+    className?: string;
+};
```

Comment:
remove types to types file and then import here

Reply:
No need, 
this type is not going to be used by any other component

Reply:
so why is separate types folder/files made, if this is the case then we don't need separate types folder because maximum time you wont be re-using the same types

Reply:
I discussed this with @sahsisunny, and we concluded what I answered